src/temp/ProjectInfo/ContactDetails.tsx

```diff
@@ -0,0 +1,90 @@
+import LocationIcon from '../icons/LocationIcon';
+import MailIcon from '../icons/MailIcon';
+import RightArrowIcon from '../icons/RightArrowIcon';
+import { ViewProfileIcon } from '../icons/ViewProfileIcon';
+import WebsiteLinkIcon from '../icons/WebsiteLinkIcon';
+import styles from './ProjectInfo.module.scss';
+import { useTranslation } from 'next-i18next';
+import SingleProjectInfoItem from './SingleProjectInfoItem';
+
+interface Props {
+  websiteURL: string;
+  location: string;
+  email: string;
+}
+
+const ContactDetails = ({ websiteURL, location, email }: Props) => {
+  const { t } = useTranslation(['donate']);
+
+  const contactDetails = [
+    {
+      icon: (
+        <ViewProfileIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: t('donate:viewProfile'),
+      link: '',
+    },
+    {
+      icon: (
+        <WebsiteLinkIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: websiteURL
+        .replace('https://www.', '')
+        .replace('http://', '')
+        .replace('https://', '')
+        .replace('https://www.', '')
+        .split(/[/?#]/)[0],
```

Comment:
URL manipulation (removing 'https://www.', 'http://', etc.) is being done directly in the component logic. It might be better to handle this manipulation outside of the component, possibly in a utility function, to improve readability and maintainability.